### PR TITLE
Updated pib-* docker images and upgrade flt cli to latest

### PR DESCRIPTION
**Pull request by kev-ms**
## Purpose of PR

- updated flt version to latest `0.6.2`
- updated docker image references for 
   - pib-imdb
   - pib-webv
   - pib-vote
   - pib-tlog
   - pib-gitops-automation

## Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [x] CI-CD changes
- [ ] GitHub Template changes

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/pib-cli/issues/337
- References https://github.com/retaildevcrews/pib-cli/issues/501 https://github.com/retaildevcrews/pib-cli/issues/502

### CODE REVIEW

File: .github/workflows/gitops.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Docker Pull
-      run: docker pull ghcr.io/bartr/goa:beta
```
Comment (bartr) [inline .github/workflows/gitops.yaml:15]: we should pull latest or maybe a specific version

**Review by bartr — APPROVED** [review]
approved with comments

File: sample-apps/vtlog/app.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 app: vtlog
 environment: dev
-image: ghcr.io/retaildevcrews/vtlog:latest
-imageName: ghcr.io/retaildevcrews/vtlog
+image: ghcr.io/cse-labs/pib-tlog:latest
+imageName: ghcr.io/cse-labs/pib-tlog
 imageTag: latest
```
Comment (aflinchb) [inline sample-apps/vtlog/app.yaml:5]: We should update the yaml in sample-apps (vtlog and bad app) to match the latest updates done in the apps for the gitops automation changes.  It doesn't have to be part of this PR, but let's at least capture it on the board 😊

**Review by aflinchb — APPROVED** [review]
Approved w/ one comment

File: sample-apps/vtlog/app.yaml (hunk 2 above)
Comment (kev-ms) [reply sample-apps/vtlog/app.yaml:5]: Captured in triage

**Pull request merged**